From af0751adc2be8caed5455c9dab0a1d5bab610567 Mon Sep 17 00:00:00 2001
From: Tk-Glitch <ti3nou@gmail.com>
Date: Tue, 15 Jan 2019 06:22:07 +0100
Subject: frog master race


diff --git a/neofetch b/neofetch
index 64c75f6e..3e10db02 100755
--- a/neofetch
+++ b/neofetch
@@ -1175,7 +1175,7 @@ get_distro() {
         distro+=" $machine_arch"
 
     [[ ${ascii_distro:-auto} == auto ]] && \
-        ascii_distro=$(trim "$distro")
+        ascii_distro="Arch"
 }
 
 get_model() {
@@ -5208,27 +5208,26 @@ EOF
         ;;

         "Arch"*)
-            set_colors 6 6 7 1
+            set_colors 2 7
             read -rd '' ascii_data <<'EOF'
-${c1}                   -`
-                  .o+`
-                 `ooo/
-                `+oooo:
-               `+oooooo:
-               -+oooooo+:
-             `/:-:++oooo+:
-            `/++++/+++++++:
-           `/++++++++++++++:
-          `/+++o${c2}oooooooo${c1}oooo/`
-${c2}         ${c1}./${c2}ooosssso++osssssso${c1}+`
-${c2}        .oossssso-````/ossssss+`
-       -osssssso.      :ssssssso.
-      :osssssss/        osssso+++.
-     /ossssssss/        +ssssooo/-
-   `/ossssso+/:-        -:/+osssso+-
-  `+sso+:-`                 `.-/+oso:
- `++:.                           `-/+/
- .`                                 `/
+${c1}       .---.`               `.---.
+    `/s${c2}yhhhy${c1}so-           -os${c2}yhhhy${c1}s/`
+   .s${c2}yNMdhNNhs${c1}s/``.---.``/s${c2}shNNhdMNy${c1}s.
+   +${c2}sdMh.`+MNss${c1}sssssssssss${c2}ssNM+`.hMds${c1}+
+   :s${c2}vNNdhNNhs${c1}sssssssssssss${c2}shNNhdNNv${c1}s:
+    /ss${c2}vhhhv${c1}sssssssssssssssss${c2}vhhhv${c1}ss/
+    .ossssssssssssssssssssssssssssso.    
+   :sssssssssssssssssssssssssssssssss:   
+  /sssssssssssssssssssssssssssssssssss/  
+ :sssssssssssssoosssssssoosssssssssssss:
+ osssssssssssss${c2}oo${c1}sssssss${c2}oo${c1}ssssssssssssso
+ osssssssssssyyyyhhhhhhhyyyyssssssssssso 
+ ${c2}\${c1}yyyyyyhhd${c2}mmmmNNNNNNNNNNNmmmm${c1}dhhyyyyyy${c2}/
+  ${c2}smmmNNNNNNNNNNNNNNNNNNNNNNNNNNNNNmmms
+   ${c2}/dNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNd/
+    ${c2}`:sdNNNNNNNNNNNNNNNNNNNNNNNNNds:`
+       ${c2}`-+shdNNNNNNNNNNNNNNNdhs+-`
+             ${c2}`.-:///////:-.`
 EOF
         ;;
 
